dual_net.py

```diff
@@ -78,6 +78,12 @@
 
 flags.DEFINE_bool('use_tpu', False, 'Whether to use TPU for training.')
 
+flags.DEFINE_bool('quantize', False, 'Whether to quantize the model.')
```

Comment:
"whether to create a quantized model" ?

(what happens if we set this flag when we load an unquantized model?)

Reply:
Presumably something bad